src/Tooltip/README.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: 'Tooltip'
 type: 'component'
+components:
+- WrapperTooltip
```

Comment:
Rather than having `WrapperTooltip` here, could we change the import to `import BaseTooltip from 'react-bootstrap/Tooltip';` and keep the component named `Tooltip` in the index.jsx file?